### ISSUE TITLE
There might be a bug in line 195 of SSG-master/reid/trainers.py

### ISSUE THREAD

**Issue by LiangY1994** [OasisYang/SSG#28]
To my understanding, should it be "loss += loss_tri" instead of "loss + loss_tri"?
